plugin-server/src/types.ts

```diff
@@ -935,3 +935,23 @@ export enum OrganizationMembershipLevel {
     Admin = 8,
     Owner = 15,
 }
+
+export interface PipelineEvent {
```

Comment:
Note: You could implement this via `extends Omit<PluginEvent, 'some' | 'other'>` and just override some/other.